content/moredata/2.md

```diff
@@ -4,25 +4,31 @@ next = "/blocksvars/1"
 prev = "/moredata/1"
 title = "Movies Schema"
 weight = 2
+endpoint = "/alter"
 +++
 
-Dgraph queries and the visualization will help us understand the schema of the movies dataset.  
+Dgraph queries, Schema Types and the visualization will help us understand the schema of the movies dataset.  
 
 As you'll see in the following pages, we can view the data from a number of perspectives; considering for example directors, movies, genres or even locations as the primary starting point.
 
 Let's have a look from the perspective of directors.  Kathryn Bigelow directed the early 90's classic Point Break.  We'll begin with her and learn about the schema.
 
-Run the query.  `_predicate_` tells us that directors have a `name` and are connected to the movies they directed via `director.film`.
+Checking the Type Schema, it tells us that directors as a "Person Type" have a `name` and are connected to the movies they directed via `director.film`.
 
 Extend the query a level deeper by also including
 ```
-director.film (first: 1) { _predicate_ }
+director.film (first: 1) {     
```

Comment:
We should query the type schema here.

Reply:
But this isn't a query. Is an edge (a nested block). Do you think I should revamp this part as a query?

Reply:
I'll merge and solve this after.